uploadNeuroDB/register_processed_data.pl

```diff
@@ -171,8 +180,9 @@
 }
 
 if  (!defined($scannerID))  {
-    print LOG "\nERROR: could not determine scannerID based on sourceFileID $sourceFileID.\n\n";
-    exit 2;
+    print LOG "\nERROR: could not determine scannerID based on sourceFileID "
+              . "$sourceFileID.\n\n";
+    exit $NeuroDB::ExitCodes::GET_SCANNERID_FROM_SOURCEFILEID_FAILURE;
```

Comment:
same comment here as above (why not use the already existing $GET_SCANNERID_FAILURE)?

Reply:
done